src/app/core/callendar/components/calendar.component.ts

```diff
@@ -157,4 +163,19 @@ export class CalendarComponent implements OnInit {
     public eventClick(event) {
         this.eventClickCallback.emit(event.detail.event.id);
     }
+
+    public eventDrop(event) {
+        this.emitWithUpdatedTime(this.eventDropCallback, event);
+    }
+
+    public eventResize(event) {
+        this.emitWithUpdatedTime(this.eventResizeCallback, event);
+    }
+
+    private emitWithUpdatedTime(eventCallback: any, event) {
+        event.detail.event.start = moment(event.detail.event.start.format());
+        event.detail.event.end = moment(event.detail.event.end.format());
+
+        eventCallback.emit(event);
```

Comment:
I feel tempted to return just this time change and session/eventId for now, instead of the whole event, or maybe we should create a more readable interface to emit.
The issue I see is that later method which reacts to this emitter can receive 'any' object but is assuming it's some kind of fullcalendar event with our session details - but that's hidden behind few children calls (event.child.child.param).
If U don't have an idea right now then we can get back to this later (maybe trello techdebt for now?)

Reply:
Agreed. We have a story about modifying the resource (Room or Judge) by dragging the session. We can improve this code when working on that story, it will give us better picture of what can/should/must be done.